src/Schema.php

```diff
@@ -121,6 +121,15 @@ private static function createOrPrependFile($pathname, $template, $separator = '
     else {
       $content = file_get_contents($pathname);
       if (FALSE === strpos($content, $template)) {
+        // Remove old content if present, looking for the template block start
+        // and end lines.
+        $templateLines = explode("\n", rtrim($template, "\n"));
+        $start = strpos($content, $templateLines[0]);
+        $end = strpos($content, end($templateLines));
+        if ($start !== FALSE && $end !== FALSE) {
+          $content = substr_replace($content, '', $start, strlen($template));
```

Comment:
I would have expected to see something in relation to `$end` here in order to remove only the amount of lines of the old/previously existing snippet…?